src/overlays/effects/ovl_Effect_Ss_Solder_Srch_Ball/z_eff_ss_solder_srch_ball.h

```diff
@@ -0,0 +1,15 @@
+#ifndef _Z_EFF_SS_SOLDERSRCHBALL_H_
+#define _Z_EFF_SS_SOLDERSRCHBALL_H_
+
+#include <ultra64.h>
+#include <global.h>
+
+typedef struct {
+    /* 0x00 */ Vec3f pos;
+    /* 0x0C */ Vec3f velocity;
+    /* 0x18 */ Vec3f accel;
+    /* 0x24 */ s16 unk_24;
+    /* 0x28 */ s16* linkDetected;
+} EffSsSolderSrchBallInitParams;
```

Comment:
The name is "Effect" not "Eff", it's just the C files for effect ss overlays don't quite have the full name.
```suggestion
} EffectSsSolderSrchBallInitParams;
```
And every function name will have to be updated accordingly as well ofc, including the one in `z_effect_soft_sprite_old_init.c`.